Kitodo/src/test/java/org/kitodo/selenium/ListingAddingST.java

```diff
@@ -49,6 +49,15 @@ public void listProjectsTest() throws Exception {
         Assert.assertEquals("Displayed wrong number of projects", numberOfProjectsInDatabase, numberOfProjectsDisplayed);
     }
 
+    @Test
+    public void listTemplatesTest() throws Exception {
+        Pages.getProjectsPage().goTo();
+        int numberOfTemplatesInDatabase = serviceManager.getTemplateService().getActiveTemplates().size();
+        int numberOfTemplatesDisplayed = Pages.getTemplatePage().countListedTemplates();
+        Thread.sleep(50000);
```

Comment:
Maybe we also dont need an extra Templates Page Class for selenium Test, as it is a tab on projects page and not a seperate page.